### PR TITLE
Wallet update integration tests

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

# Overview

- [ ] I have added tests for wallet update name

# Comments
Still working on wallet update pass integration tests. Can get them in this PR (or another one if this one gets reviewed/approved before I finish)

### CODE REVIEW

File: lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs [hunk 1]
```diff
@@ -830,6 +830,196 @@ spec = do
...
+        ru <- request @ApiWallet ctx ("PUT", "v2/wallets" </> walId) Default newName
+        verify ru expectations
+        rg <- request @ApiWallet ctx ("GET", "v2/wallets" </> walId) Default Empty
+        verify rg expectations
+        rl <- request @[ApiWallet] ctx ("GET", "v2/wallets") Default Empty
+        verify rl
+            [ expectResponseCode @IO HTTP.status200
+            , expectListSizeEqual 1
+            , expectListItemFieldEqual 0 walletName "New great name"
+            , expectListItemFieldEqual 0 addressPoolGap 20
```
Comment (rvl) [inline lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs:859]: The verifications could be shortened to just check walletName and walletId without reducing the effectiveness of this test.

File: lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs [hunk 2]
```diff
@@ -875,6 +1065,11 @@ spec = do
             "passphrase": "Secure passphrase"
             } |]
 
+    updateNamePayload :: Text -> Payload
+    updateNamePayload name = Json [json| {
```
Comment (rvl) [inline lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs:1069]: This is also a possibility:
```
Json (Object ("name" .= name))
```

**Review by rvl — APPROVED** [review]
Looks good.

File: lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs (hunk 2 above)
Comment (piotr-iohk) [reply lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs:1069]: thanks for the hint :)

File: lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs (hunk 1 above)
Comment (piotr-iohk) [reply lib/http-bridge/test/integration/Test/Integration/Scenario/Wallets.hs:859]: Yes, most probably. Although in general I prefer to _verify_ as much as possible whenever I can (just to be safe).

**Pull request merged**